app/pages/project/translations.js

```diff
@@ -0,0 +1,30 @@
+import apiClient from 'panoptes-client/lib/api-client';
+import counterpart from 'counterpart';
+
+counterpart.setFallbackLocale('en');
+
+
+const translations = {
```

Comment:
This all works, but should probably be replaced with a Redux store, and `load` action, at some point.